cargo-shuttle/src/lib.rs

```diff
@@ -1004,10 +991,12 @@ mod tests {
     use std::path::PathBuf;
     use std::str::FromStr;
 
-    fn path_from_workspace_root(path: &str) -> PathBuf {
+    pub fn path_from_workspace_root(path: &str) -> PathBuf {
         PathBuf::from(std::env::var("CARGO_MANIFEST_DIR").unwrap())
             .join("..")
             .join(path)
+            .canonicalize()
```

Comment:
I'm not certain this would be an issue, but should we use `dunce::canonicalize` to be safe on windows?